### PR TITLE
News page

### LINKED ISSUE #12: News page
**Figma design**
image

**Figma prototype**
[Link to Figma prototype](https://www.figma.com/file/NF91V1N9blvG6slT68Z5bx/Our-SK?node-id=1096%3A2618)

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
image

**Features**
- News page
  Resolves #12
- News Card component
- Current Period Indicator in Schedule
- Teacher in Home has scrollbar again
- More readable Subject List header ratio

**Fixes**
- Class Advisor image is too rounded

### CODE REVIEW

File: src/pages/news.tsx [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        "en-US": {
+          title: "Classes Feedback",
+          supportingText:
+            "Before every Monday, all students must test for COVID-19 via RT-PCR, ATK, or other trusted methods and submit \
+            their readings here.",
+        },
+        th: {
+          title: "บันทึกผลการตรวจโควิด-19",
+          supportingText:
+            "ก่อนทุกวันจันทร์ ให้นักเรียนถ่ายรูปผลการตรวจการหาเชื้อโควิด-19 ผ่าน RT-CPR ATK หรือวิธีอื่นๆ",
```
Comment (IHasDiabetes) [inline src/pages/news.tsx:183]: Not sure if this is relevant but this should be RT-PCR

**Review by Jimmy-Tempest — APPROVED** [review]
LGTM!

**Review by IHasDiabetes — APPROVED** [review]
LGTM!! :D

File: src/pages/news.tsx (hunk 1 above)
Comment (SiravitPhokeed) [reply src/pages/news.tsx:183]: Fun fact, this was copied from News in Draft 3, which was copied from Home in Draft 3, which was copied from Home in Draft 2, which was copied from Home in Draft 1. No one noticed, until now! 🎉

Comment (Jimmy-Tempest) [reply src/pages/news.tsx:183]: Great

**Pull request merged**